scripts/Modules/TTK_config.py

```diff
@@ -0,0 +1,24 @@
+import mkw_translations as trans
+
+# uses R button instead of B button to drift when enabled
+useRbutton = True
+
+# creates up to backupAmount backup files for your race inputs while you are TASing
+# set to 0 to disable backups
+backupAmount = 0
+
+textFilePath = {
+    "player": "MKW_Inputs\\MKW_Player_Inputs_" + trans.course_slot_abbreviation() + ".csv",
```

Comment:
Shouldn't you use os.path.join to do path separators for you?

Reply:
Added this changed in 👍